cdap-docs/admin-manual/source/installation/security.rst

```diff
@@ -270,43 +286,58 @@ LDAP Authentication
+``security.authentication.handler.bindPassword``           ``<bindPassword>``          The password of the user on an
+                                                                                       external LDAP server permitted to 
+                                                                                       search the LDAP directory
+``security.authentication.handler.userIdAttribute``        ``<userIdAttribute>``       The Unique User ID Attribute
+``security.authentication.handler.userPasswordAttribute``  ``<userPasswordAttribute>`` Password of the ``userIdAttribute``
+``security.authentication.handler.roleBaseDn``             ``<roleBaseDn>``            Role Based: <to be confirmed>
+``security.authentication.handler.roleNameAttribute``      ``<roleNameAttribute>``     Name of attribute specifying the role 
+                                                                                       <to be confirmed>
+``security.authentication.handler.roleMemberAttribute``    ``<roleMemberAttribute>``   Name of attribute specifying the role 
+                                                                                       <to be confirmed>
+``security.authentication.handler.roleObjectClass``        ``<roleObjectClass>``       Name of attribute specifying the role 
+                                                                                       <to be confirmed>
+========================================================== =========================== ======================================
 
 To enable SSL between the authentication server and the LDAP instance, configure
 these properties in ``cdap-site.xml``:
 
-========================================================== ================= ================================================
-Property                                                   Value             Default Value
-========================================================== ================= ================================================
-``security.authentication.handler.useLdaps``               ``true/false``    ``false``
-``security.authentication.handler.ldapsVerifyCertificate`` ``true/false``    ``true``
-========================================================== ================= ================================================
+========================================================== ================= ========= ======================================
+Property                                                   Default Value     Value     Description
+========================================================== ================= ========= ======================================
+``security.authentication.handler.useLdaps``               ``false``         ``true``  Set to ``true`` to enable use of LDAP
+``security.authentication.handler.ldapsVerifyCertificate`` ``true``          ``true``  Set to ``true`` to enable SSL between
```

Comment:
Set to ``true`` to enable verification of the SSL certificate used by the LDAP server